StyleCop.Analyzers/StyleCop.Analyzers/Helpers/XmlCommentHelper.cs

```diff
@@ -199,6 +217,24 @@ internal static string GetText(XmlTextSyntax textElement, bool normalizeWhitespa
             return result;
         }
 
+        internal static T GetAttribute<T>(XmlNodeSyntax nodeSyntax) where T : XmlAttributeSyntax
```

Comment:
:exclamation: More than one such attribute can exist. To make the behavior of this method more clear, please do one of the following (or feel free to propose an alternative solution):

1. Rename the method to indicate the behavior, such as `GetFirstAttributeOrDefault<T>`.
2. Change the method to return an `IEnumerable<T>` instead of `T`.